features/steal-game/script.js

```diff
@@ -0,0 +1,18 @@
+var enabledStealGame = true;
+
+if (window.location.href.startsWith("https://scratch.mit.edu/projects/")) {
+    if (enabledStealGame) {
+        var remixButton = document.querySelector('.remix-button')

```

Comment:
I'm not sure why this variable exists- it isn't needed, since there aren't any asynchronous functions.

Reply:
Ah sorry, that was just me trying to find out how to get dynamic enable to work and I forgot to remove it